### PR TITLE
SDK Troubleshooting Doc

### PR DESCRIPTION
**Pull request by brittBarak**
added sdk troubleshooting doc

### CODE REVIEW

File: _documentation/client-sdk/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+navigation_weight: 1
+---
+
+# Troubleshooting
+
+## Working with the Nexmo CLI
+
+### Nexmo Application setup
+
+Since you can create multiple Nexmo applications, the commands you run reffer to the application that was set up. For example, when you create a user, you must make sure to create it on the application you intended.
```
Comment (tbedford) [inline _documentation/client-sdk/troubleshooting.md]: reffer -> refer

File: _documentation/client-sdk/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+     for example:
+
+     ```sh
+    nexmo app:setup abababa-ababa-123123-123123-abababa private.key
+    ```
+
+### No response to commands
+
+It you run a command and don't get a response
+    * Try making sure that all the json objects in you command are colsed objects, and not missing any `}` or `'` for example.
```
Comment (tbedford) [inline _documentation/client-sdk/troubleshooting.md]: json -> JSON

File: _documentation/client-sdk/troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+## JWTs
+
+> Remember that a JWT is per user per Nexmo Application
+
+### Invalid Token Error
+
+* [Decode your JWT](https://jwt.io/)
+
+* Make sure the `"application_id"` is correct
```
Comment (tbedford) [inline _documentation/client-sdk/troubleshooting.md]: Why backticks and quotes?

File: _documentation/client-sdk/troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+* Make sure the `“sub”` is correct. Meaning, a user with this user name exists in your Nexmo Application.
+
+* Make sure the JWT isn't expired.
```
Comment (tbedford) [inline _documentation/client-sdk/troubleshooting.md]: Make sure the JWT isn't expired. -> has not expired.

File: _documentation/client-sdk/troubleshooting.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+    * You can find the expiration date on `“exp”`, in Unix time, which is seconds since Jan 01 1970(UTC).
+    
+    * You can [convert it to human time](https://www.epochconverter.com/).
+    
+    * Make sure the expiration time is the future, meaning the jwt hasnt expired yet.
+
+### Connection error or Connection Timeout 
+
+Getting Connection error or Connection Timeout while trying to login to the SDK
```
Comment (tbedford) [inline _documentation/client-sdk/troubleshooting.md]: Generally before a list or code break terminate the line with a colon.

**Comment by brittBarak** [comment]
@tbedford  you're the best! thank you!

File: _documentation/client-sdk/troubleshooting.md (hunk 3 above)
Comment (brittBarak) [reply _documentation/client-sdk/troubleshooting.md]: since this is how it appears on the JWT, with quotes, so it made more sense to me.. no? 🤔

Comment (tbedford) [reply _documentation/client-sdk/troubleshooting.md]: OK that's fine then.

File: _documentation/client-sdk/troubleshooting.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+     for example:
+
+     ```sh
+    nexmo app:setup abababa-ababa-123123-123123-abababa private.key
+    ```
+
+### No response to commands
+
+It you run a command and don't get a response
+    * Try making sure that all the JSON objects in you command are colsed objects, and not missing any `}` or `'` for example.
```
Comment (lornajane) [inline _documentation/client-sdk/troubleshooting.md]: Should "colsed" be "closed" ? (sorry just saw it in an email notification and couldn't unsee it!)

Comment (brittBarak) [reply _documentation/client-sdk/troubleshooting.md]: How come VS Code doesn't have spell checker?? 
Thank you Lorna!!!

File: _documentation/client-sdk/troubleshooting.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+* Make sure the `“sub”` is correct. Meaning, a user with this user name exists in your Nexmo Application.
+
+* Make sure the JWT hasn't expired.
+
+    * You can find the expiration date on `“exp”`, in Unix time, which is seconds since Jan 01 1970(UTC).
+    
+    * You can [convert it to human time](https://www.epochconverter.com/).
+    
+    * Make sure the expiration time is the future, meaning the jwt hasnt expired yet.
```
Comment (tbedford) [inline _documentation/client-sdk/troubleshooting.md]: jwt -> JWT. hasnt -> hasn't or has not.

File: _documentation/client-sdk/troubleshooting.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+## Push notifications
+
+* Make sure you’ve uploaded the certificate to Nexmo's server.
+
+* Make sure that you've enabled push notifications, and that the method `client.enablePushNotifications()` was successful. You might even want to put a log or a break point to ensure that.
+
+* You need to have a valid admin `JWT`, meaning a `JWT` without “sub” claim. You can [Decode your JWT](https://jwt.io/) to make sure.
+
+
+# Have more Questions?
```
Comment (tbedford) [inline _documentation/client-sdk/troubleshooting.md]: Extra blank line after heading.

**Pull request merged**